app/controllers/api/v1/reactions_controller.rb

```diff
@@ -42,12 +43,20 @@ def toggle
 
       private
 
+      def pundit_user
+        (current_user || @user)
+      end
```

Comment:
**Question:** What is the purpose of having this method? If we need it, should it live in the V1 `api_controller` instead so it's available to all controllers?

I'm asking because I commented it out locally and all API specs pass without it (ran it with `bundle exec rspec spec/requests/api/i`).

I remember that on v0 there was some work done spearheaded by Jeremy at the time to rework a bit how our policies are structured, which is why we have that on V0's `api_controller`, but I'm unsure I understand the reasoning behind having this method here 🙂

Reply:
Great catch! I recall I did have a test fail early on without this, but I later figured out a better way to set up the test user, and did not revisit this.